tests/program_analysis/test_py_ast_to_cast.py

```diff
@@ -0,0 +1,846 @@
+
+    filepath = f"{DATA_DIR}/{folder}/{prog_name}"
```

Comment:
@titomeister the following lines `78 -- 93` seem to be repeated from the code above. Could you create a helper function that both of these tests call that holds this code?